python/lsst/afw/cameraGeom/utils.py

```diff
@@ -106,12 +106,12 @@ def plotFocalPlane(camera, fieldSizeDeg_x=0, fieldSizeDeg_y=None, dx=0.1, dy=0.1
     xs = []
     ys = []
     pcolors = []
-    for pos in zip(field_gridx, field_gridy):
-        # pos is a pair of lsst.afw.geom.Angle
-        # but the units of FIELD_ANGLE are radians
-        posFieldAngle = afwGeom.Point2D(math.radians(pos[0]), math.radians(pos[1]))
-        fieldAngleToFocalPlane = camera.getTransform(FIELD_ANGLE, FOCAL_PLANE)
-        posFocalPlane = fieldAngleToFocalPlane.applyForward(posFieldAngle)
+
+    # compute focal plane positions corresponding to field angles field_gridx, field_gridy
+    posFieldAngleRadiansList = [afwGeom.Point2D(x.asRadians(), y.asRadians())
+                                for x, y in zip(field_gridx, field_gridy)]
+    posFocalPlaneList = camera.transform(posFieldAngleRadiansList, FIELD_ANGLE, FOCAL_PLANE)
```

Comment:
Why isn't `std::pair<Angle, Angle>` not a supported endpoint type?

Reply:
I think that adds too much complexity, especially because it also requires adding support for `std::vector<std::pair<Angle, Angle>>` and adding a new Endpoint type. In any case it is clearly out of scope for this ticket.